cs/test/core/BondedTests.cs

```diff
@@ -438,5 +438,15 @@ public void SerializeBonded()
             Assert.IsTrue(Comparer.Equal(poly1, obj.poly[1].Deserialize()));
             Assert.IsTrue(Comparer.Equal(poly2, obj.poly[2].Deserialize<Derived>()));
         }
+
+        [Test]
```

Comment:
Minor: in the first commit, consider adding `[Ignore("Intentionally failing CBv2 regression test before bug fix")]` to this test, as it's known broken. That way, the build and tests still "pass".